pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,459 @@
+    echo "${OPTION}=O_DIRECT_NO_FSYNC" >> ${OUTPUT_FILE}
+    echo "${OPTION}=littlesync" >> ${OUTPUT_FILE}
+    echo "${OPTION}=nosync" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-checksum-algorithm" ]; then
+    echoit "  > Adding possible values none, innodb, crc32 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=innodb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=crc32" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-disable" -o "${OPTION}" == "--innodb-monitor-enable" -o "${OPTION}" == "--innodb-monitor-reset" -o "${OPTION}" == "--innodb-monitor-reset-all" ]; then
+    echoit "  > Adding possible values counter, module, pattern and all for option '${OPTION}' to the final list..."
+    echo "${OPTION}=counter" >> ${OUTPUT_FILE}
+    echo "${OPTION}=module" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pattern" >> ${OUTPUT_FILE}
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-stats-method" ]; then
+    echoit "  > Adding possible values nulls_equal, nulls_unequal, nulls_ignored for option '${OPTION}' to the final list..."
+    echo "${OPTION}=nulls_equal" >> ${OUTPUT_FILE}
+    echo "${OPTION}=nulls_unequal" >> ${OUTPUT_FILE}
+    echo "${OPTION}=nulls_ignored" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-bin" ]; then
+    echoit "  > Adding possible value binlog for option '${OPTION}' to the final list..."
+    echo "${OPTION}=binlog" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-slow-rate-type" ]; then
+    echoit "  > Adding possible values session, query for option '${OPTION}' to the final list..."  
+    echo "${OPTION}=session" >> ${OUTPUT_FILE}
+    echo "${OPTION}=query" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--replica-type-conversions" ]; then
+    echoit "  > Adding possible values ALL_LOSSY, ALL_NON_LOSSY, ALL_SIGNED, ALL_UNSIGNED for option '${OPTION}' to the final list..."
+    echo "${OPTION}=ALL_LOSSY" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_NON_LOSSY" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_SIGNED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_UNSIGNED" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--performance-schema-accounts-size" -o "${OPTION}" == "--performance-schema-hosts-size" -o "${OPTION}" == "--performance-schema-max-thread-instances" -o "${OPTION}" == "--performance-schema-users-size" ]; then  
+    echoit "  > Adding possible values 0, 1, 2, 12, 24, 254, 1023, 2047, 1048576 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=12" >> ${OUTPUT_FILE}
+    echo "${OPTION}=24" >> ${OUTPUT_FILE}
+    echo "${OPTION}=254" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1023" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2047" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1048576" >> ${OUTPUT_FILE}
+    echoit "  > Adding possible values relay-bin for option '${OPTION}' to the final list..."
```

Comment:
You have deleted the elif condition accidentally here :(

Reply:
fixed :)